app/src/main/res/layout/activity_settings.xml

```diff
@@ -205,6 +206,21 @@
 
             </RelativeLayout>
 
+            <RelativeLayout
+                android:id="@+id/settings_enable_capitalization_row_holder"
```

Comment:
stick to the casual style please, there should be no "row" and rename the views after reverting the string change below.